src/services/completions.ts

```diff
@@ -910,6 +910,9 @@ namespace ts.Completions {
                     case SyntaxKind.PropertyAccessExpression:
                         propertyAccessToConvert = parent as PropertyAccessExpression;
                         node = propertyAccessToConvert.expression;
+                        if (isMetaProperty(node)) {

```

Comment:
This is incorrect change since this will return `undefined` for scenario that @DanielRosenwasser pointed out.. (that is when ImportMeta interface has some properties defined.)

Reply:
ah I see. I changed it so that it would correctly handle cases where ImportMeta has properties defined and added a test case.